playwright.config.ts

```diff
@@ -85,7 +85,7 @@ const config: PlaywrightTestConfig = {
   maxFailures: headless ? 10 : undefined,
   fullyParallel: true,
   reporter: [
-    [process.env.CI ? "github" : "list"],
+    [process.env.CI ? "blob" : "list"],
```

Comment:
the GitHub reporter is not useful since we're using matrix strategies. https://playwright.dev/docs/test-reporters#github-actions-annotations